website/docs/guides/airflow-quickstart.md

```diff
@@ -1,352 +1,337 @@
 ---
 sidebar_position: 2
 ---
-# Getting Started with Airflow and OpenLineage+Marquez
 
-In this example, we'll walk you through how to enable Airflow DAGs to send lineage metadata to [Marquez](https://marquezproject.ai/) using OpenLineage. 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
 
-### You’ll Learn How To:
+# Getting Started with Apache Airflow® and OpenLineage+Marquez
 
-* configure Airflow to send OpenLineage events to Marquez
-* write OpenLineage-enabled DAGs
-* troubleshoot a failing DAG using Marquez
+In this tutorial, you'll configure Apache Airflow® to send OpenLineage events to [Marquez](https://marquezproject.ai/) and explore a realistic troubleshooting scenario. 
 
 ### Table of Contents
 
-1. [Step 1: Configure Your Astro Project](#configure-your-astro-project)
-2. [Step 2: Add Marquez Services Using Docker Compose](#add-marquez-services-using-docker-compose)
-3. [Step 3: Start Airflow with Marquez](#start-airflow-with-marquez)
-4. [Step 4: Write Airflow DAGs](#write-airflow-dags)
-5. [Step 5: View Collected Metadata](#view-collected-metadata)
-6. [Step 6: Troubleshoot a Failing DAG with Marquez](#troubleshoot-a-failing-dag-with-marquez)
+1. [Prerequisites](#prerequisites)
+2. [Get and start Marquez](#get-marquez)
+3. [Configure Apache Airflow to send OpenLineage events to Marquez](#configure-airflow)
+4. [Write Airflow DAGs](#write-airflow-dags)
+5. [View Collected Lineage in Marquez](#view-collected-metadata)
+6. [Troubleshoot a Failing DAG with Marquez](#troubleshoot-a-failing-dag-with-marquez)
 
-## Prerequisites
+## Prerequisites {#prerequisites}
 
 Before you begin, make sure you have installed:
 
 * [Docker 17.05](https://docs.docker.com/install)+
-* [Astro CLI](https://docs.astronomer.io/astro/cli/overview)
-* [curl](https://curl.se/)
+* [Apache Airflow 2.8+](https://airflow.apache.org/docs/apache-airflow/stable/start.html) running locally.
```

Comment:
I think the new provider works from 2.7+

Reply:
https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-7-0-2023-08-18
shows up in the release notes, so I'm remembering correctly. but maybe there's a reason to direct people to use 2.7? @mobuchowski ?

Reply:
Airflow periodically bumps minimum required Airflow version for all providers, that's why we have 2.8 for the latest OL provider. Last version of OL provider to support Airflow 2.7 is [1.10](https://airflow.apache.org/docs/apache-airflow-providers-openlineage/1.10.0/index.html#requirements). We can say 2.7 in the doc as it's possible to use the provider with this version, it would just be outdated.

Reply:
Changed it to 2.7 in light of @kacpermuda 's comment

Reply:
Changed to 2.7+ in light of @kacpermuda 's comment